src/main/java/uk/gov/hmcts/darts/authentication/component/impl/TokenValidatorImpl.java

```diff
@@ -52,16 +50,16 @@ public JwtValidationResult validate(String accessToken, AuthProviderConfiguratio
                 JWTClaimNames.EXPIRATION_TIME,
                 JWTClaimNames.ISSUED_AT,
                 JWTClaimNames.SUBJECT,
-                EMAILS_CLAIM_NAME
+                configuration.getClaims()
             ))
         );
         jwtProcessor.setJWTClaimsSetVerifier(claimsVerifier);
 
         try {
-            JWTClaimsSet claimsSet = jwtProcessor.process(accessToken, null);
-            log.debug("Validation successful - emailAddresses: {}", claimsSet.getStringListClaim(EMAILS_CLAIM_NAME));
```

Comment:
I found this debug useful - for the external B2C_1_darts_externaluser_signin | Application claims - we eventually get an access token with the 'emails' claim. This corresponds to my preferred username for the Portal login.
So I'm wondering about our TABLE user_account `user_email_address` - I'm expecting that column to be NOT NULL and what about `user_name` column, we have a claim called `name`, would you expect that to map across at some point?

Reply:
We should not be writing claims on log file, thats Security violation. Any claims (emails or prefered_username) will map to user_email_address in user_account Table.